jobs/categorizeConversationToIssueGroup.ts

```
@@ -0,0 +1,197 @@
+import { eq, inArray } from "drizzle-orm";
+import { z } from "zod";
+import { db } from "@/db/client";
+import { conversations } from "@/db/schema/conversations";
```

ESLint

[error] Replace `s·}·from·"@/db/schema/conversation` with `Messages·}·from·"@/db/schema/conversationMessage`

(prettier/prettier)